app/src/main/res/values/strings.xml

```diff
@@ -7,7 +7,8 @@
     <string name="password_hint"><i>Password</i></string>
     <string name="code_hint"><i>Authorization Code</i></string>
     <string name="sign_in">Sign In</string>
-    <string name="sign_in_as_guest">Sign In As Guest</string>
+    <string name="sign_in_as_guest">SIGN IN AS GUEST</string>
```

Comment:
Why change it from lowercase to uppercase when the Button View converts it to uppercase anyway? xD
(Just a remark; is ok to leave it like this)

Reply:
bc i converted the button to a textview and then back to button 💃